airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -99,8 +101,15 @@ def convert_schema_into_query(
         query_template = f"SELECT {fields} FROM {from_category} "
 
         if cursor_field:
-            # Fix issue 5411: Make date_start and date_end inclusive.
-            query_template += f"WHERE {cursor_field} >= '{from_date}' AND {cursor_field} <= '{to_date}' ORDER BY {cursor_field} ASC"
+            if from_category == 'click_view':
+                # ClickView must have a filter limiting the results to one day.
+                from_date = pendulum.parse(from_date)
+                to_date = from_date.add(days=1).to_date_string()
+                from_date = from_date.to_date_string()
+                query_template += f"WHERE {cursor_field} > '{from_date}' AND {cursor_field} <= '{to_date}' ORDER BY {cursor_field} ASC"
+            else:
+                # Fix issue 5411: Make date_start and date_end inclusive.
+                query_template += f"WHERE {cursor_field} >= '{from_date}' AND {cursor_field} <= '{to_date}' ORDER BY {cursor_field} ASC"
```

Comment:
I think you can refactor your code to looks simpler:
```python
if from_category == 'click_view':
    # ClickView must have a filter limiting the results to one day.
    to_date = pendulum.parse(from_date).add(days=1).to_date_string()

query_template += f"WHERE {cursor_field} > '{from_date}' AND {cursor_field} <= '{to_date}' ORDER BY {cursor_field} ASC"
```

Reply:
resolved